[feature] #1638: `configuration` return doc subtree (#2291)
hyperledger/iroha: docs/source/references/api_spec.md

**Expects**:

> There are 2 variants:

Reviewer: ```suggestion
There are two options:
```

> - It either expects json body `"Value"` and returns configuration value as json

Reviewer: ```suggestion
- Expects: a JSON body `"Value"`. Returns: a configuration value as JSON.
```

> - Or it expects json body like below and returns documentation for specific field (as json string) or null. If the requested

Reviewer: ```suggestion
- Expects: a JSON body that specifies the field (see example below). Returns: documentation for a specific field (as JSON string) or `null`.

Note that if the requested
```
